templates/deployments.yaml

```diff
@@ -862,9 +871,7 @@ spec:
         checksum/config: {{ include (print $.Template.BasePath "/configmaps_st2-conf.yaml") $ | sha256sum }}
         checksum/packs: {{ include (print $.Template.BasePath "/configmaps_packs.yaml") $ | sha256sum }}
         checksum/datastore-key: {{ include (print $.Template.BasePath "/secrets_datastore_crypto_key.yaml") $ | sha256sum }}
-        {{- if .annotations }}
-{{ toYaml .annotations | indent 8 }}
-        {{- end }}
+{{ include "custom-annotations" . | indent 8 }}
```

Comment:
why?

Reply:
I inlined the template. I'm fine either way, but I'm still curious why you prefer that.

Reply:
Just trying to keep the balance and avoid the level of indirection when jumping between the files and templates.

Templating makes really sense when more advanced logic is happening under the hood.

Otherwise, everything could be templated from the Pod/Deployment spec which will reduce the number of lines from the one side, but make the spec harder to follow.